charts/kube-prometheus-stack/Chart.yaml

```
@@ -21,7 +21,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 45.4.0
+version: 45.4.1
```

ast the feaurtre version of the subchart changed the feature version of the main chart should change too.
```suggestion
version: 45.5.0
```

---

Thank you!